cartographer/kalman_filter/unscented_kalman_filter.h

```diff
@@ -158,7 +158,7 @@ class UnscentedKalmanFilter {
     W.reserve(2 * N + 1);
     W.emplace_back(StateType::Zero());
 
-    std::vector<Eigen::Matrix<FloatType, K, 1>> Z;
+    std::vector<Eigen::Matrix<FloatType, K, 1>, Eigen::aligned_allocator<Eigen::Matrix<FloatType, K, 1>>> Z;
```

Comment:
Similar here: Rather add Eigen::DontAlign to the matrix type, verify that this is not causing performance degradation, and solve it locally.